apps/aecore/lib/aecore/chain/identifier.ex

```diff
@@ -30,25 +30,7 @@ defmodule Aecore.Chain.Identifier do
   @spec create_identity(type(), value()) :: Identifier.t() | {:error, String.t()}
   # byte_size(data) == 32 data should be stricted to 32 bytes only
   def create_identity(value, type) when is_atom(type) and is_binary(value) do
```

Comment:
So now if we pass wrong data to this function, it is supposed to crash with guards expression? And what happens when we add guard for value being only 32 bytes?